core/src/main/scala/org/apache/spark/MapOutputTracker.scala

```diff
@@ -121,12 +121,28 @@ private class ShuffleStatus(numPartitions: Int) {
     mapStatuses(mapIndex) = status
   }
 
+  /**
+   * Update the map output location (e.g. during migration).
+   */
+  def updateMapOutput(mapId: Long, bmAddress: BlockManagerId): Unit = withWriteLock {
+    val mapStatusOpt = mapStatuses.find(_.mapId == mapId)
+    mapStatusOpt match {
+      case Some(mapStatus) =>
+        logInfo("Updating map output for ${mapId} to ${bmAddress}")
```

Comment:
String interpolation missing:

```suggestion
        logInfo(s"Updating map output for ${mapId} to ${bmAddress}")
```